src/storage/backup/src/lib.rs

```diff
@@ -118,3 +121,26 @@ impl From<&MetaSnapshotManifest> for PbMetaSnapshotManifest {
         }
     }
 }
+
+const CHECKPOINT_FILE_NAME: &str = "checkpoint";
+
+/// Creates the object store to persist checkpoint, using the same object store url with
+/// `state_store`.
+pub async fn object_store_client(state_store_url: &str) -> ObjectStoreImpl {
+    let url = match state_store_url {
+        hummock if hummock.starts_with("hummock+") => {
+            hummock.strip_prefix("hummock+").unwrap().to_string()
+        }
+        _ => "memory".to_string(),
+    };
+    parse_remote_object_store(
+        &url,
+        Arc::new(ObjectStoreMetrics::unused()),
```

Comment:
How about support passing ObjectStoreMetrics in `fn object_store_client`? It is useful to also emit object store metrics in meta node.